### PR TITLE
[DDW-916] Update second message on the initial screen to in include part regarding the waiting time

### PR DESCRIPTION
**Pull request by szymonmaslowski**
This PR changes the second message on the loading screen removing the waiting time information from it.

## Screenshots

image
image

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1642710744099400)

---

### Basics
- [X] PR assigned to the PR author(s) 
- [X] `input-output-hk/daedalus-dev` and `input-output-hk/daedalus-qa` assigned as PR reviewers
- [X] If there are UI changes, Alexander Rukin assigned as an additional reviewer
- [X] PR has appropriate labels (`release-vNext`, `feature`/`bug`/`chore`, `WIP`)
- [X] PR link is added to a Jira ticket, ticket moved to In Review
- [X] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [X] PR has a good description that summarizes all changes
- [X] PR contains screenshots (in case of UI changes)
- [X] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [X] There are no missing translations (running `yarn manage:translations` produces no changes)
- [X] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [X] Japanese text changes are proofread and approved (Junko Oda)
- [X] Storybook works and no stories are broken (`yarn storybook`)
- [X] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Review by danielmain — APPROVED** [review]

**Review by gabriela-ponce — APPROVED** [review]
@szymonmaslowski The second message looks good 🎉 
Just a small comment we probably missed on the previous PR, there is an extra dot in the ellipsis from the first message. Check [this](https://drive.google.com/file/d/1tHku6g2ebTPXUMKCKtmui4lGH9pTkGYZ/view) video.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ### Features
 
+- Updated message explaining the node syncing process on the initial screen ([PR 2831](https://github.com/input-output-hk/daedalus/pull/2831))
```
Comment (mchappell) [inline CHANGELOG.md:7]: These two lines should be merged, please see the rest of changelog for examples

**Review by mchappell — APPROVED** [review]

**Review by alexander-rukin — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (szymonmaslowski) [reply CHANGELOG.md]: Done :+1:

**Comment by szymonmaslowski** [comment]
> @szymonmaslowski The second message looks good 🎉 Just a small comment we probably missed on the previous PR, there is an extra dot in the ellipsis from the first message. Check [this](https://drive.google.com/file/d/1tHku6g2ebTPXUMKCKtmui4lGH9pTkGYZ/view) video.

Thanks Gabriela for catching that. Fix for that is already pushed :+1:

**Pull request merged**